doc/engines/libfuzzer.rst

```diff
@@ -0,0 +1,54 @@
+*********
+Libfuzzer
+*********
+
+Libfuzzer works in a different way than the other engines. It is not a standalone fuzzer binary
+but the whole fuzzing logic is embedded in the target binary. Thus there is no installation steps
+required.
+
+
+Running pastis-libfuzzer
+========================
+
+The program ``pastis-libfuzzer`` wraps all interaction with the broker.
+It can either be launched locally in offline mode or with the broker in
+an online mode.
+
+Online mode
+-----------
+
+Launching the ``pastis-libfuzzer`` in online mode is as simple as:
+
+.. code-block:: bash
+
+    $ pastis-aflpp online
```

Comment:
To be honest I haven't tried it myself but it looks like a copy-paste error

Reply:
True! Just fixed it: https://github.com/quarkslab/pastis/pull/23/commits/980e03092f2022a930a6695ca585462fcf13a911